shared/wallets/wallet/settings-popup/index.js

```diff
@@ -0,0 +1,188 @@
+// @flow
+import * as React from 'react'
+import {
+  Avatar,
+  Box2,
+  ClickableBox,
+  Dropdown,
+  Icon,
+  iconCastPlatformStyles,
+  Text,
+} from '../../../common-adapters'
+import {collapseStyles, globalColors, globalMargins, globalStyles, platformStyles, styleSheetCreate} from '../../../styles'
+
+export type Props = {
+  name: string,
+  user: string,
+  isDefault: boolean,
+  currency: string,
+  currencies: Array<string>,
+  onDelete: () => void,
+  onSetDefault: () => void,
+  onEditName: () => void,
+  onCurrencyChange: (currency: string) => void,
+}
+
+const makeDropdownItems = (props: Props) => {
+  const items = [
+    <Box2 centerChildren={true} direction="vertical" key="_header">
+      <Text type="BodySmall" style={styles.dropdownHeader}>
+        Past transactions won't be affected by this change.
+      </Text>
+    </Box2>,
+  ]
+  return items.concat(props.currencies.map(s => makeDropdownItem(s, s === props.currency)))
+}
+
+const makeDropdownItem = (item: string, isSelected: boolean) => {
```

Comment:
since this function is a single return, it can use the shorthand
```
const makeDropdownItem = (item: string, isSelected: boolean) => (
  <Box2 ...>
  </Box2>
)
```
but it's a style thing so up to you.

Technically, so can `SettingsPopup`, but it's long enough that there's a good chance it'll need more than just the return in the future.